docs/website/src/main/resources/certifications/jakarta-connectors/2.1/TCK-Results.md

```diff
@@ -0,0 +1,183 @@
+TCK Results
+===========
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta Connectors.
+
+# 2.1 Certification Request
+
+- [x] Organization Name ("Organization") and, if applicable, URL: <br/>
+  Eclipse Foundation
+- [x] Product Name, Version and download URL (if applicable): <br/>
+  [GlassFish 7.0.0-M4](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/7.0.0-M4/glassfish-7.0.0-M4.zip)
+- [x] Specification Name, Version and download URL: <br/>
+  [Jakarta Connectors 2.1](https://jakarta.ee/specifications/connectors/2.1/)
+- [x] TCK Version, digital SHA-256 fingerprint and download URL: <br/>
+  [Jakarta Connectors TCK 2.1.0](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-connectors-tck-2.1.0.zip), SHA-256: 6b044e8361e1fd6e5ab46f3dfc3850332b23a192a1b872f878b3380aa9561bcc
```

Comment:
SHA value should be: f8c018fe56f19899a780df3f8d10c93555996861038b7da03eee5421574d4b9b

Reply:
it was changed again ( [jakarta-connectors-tck-2.1.0-tckinfo.txt](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-connectors-tck-2.1.0-tckinfo.txt) ). Would it be a good idea to append a download date until the final version of TCK is released?

Reply:
The important thing with regards to ratifying the spec and claiming compatibility for an implementation is that it is the one and only TCK that is used for the CCR. It has to be the same SHA that is published on the website. A CCR with a different SHA would not be valid.

Reply:
its recommended to use promoted tck (https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/promoted/eftl/) for certification and ballot purpose, promotion can be done by sending mail to tck-dev(post stable tck results are got).
I have disabled the connector TCK build at CI, so the new connector TCK is not staged.